BlogTemplate.Tests/Fakes/FakeFileSystem.cs

```diff
@@ -113,7 +113,15 @@ void IFileSystem.WriteFileText(string path, string text)
 
         void IFileSystem.AppendFile(string path, byte[] data)
         {
-            
+            if (!_files.ContainsKey(path))
+            {
+                AddFile(path);
+            }
+
+            _files[path].Seek(0, SeekOrigin.Begin);

```

Comment:
This method could just call the next overload and pass in an appropriate offset (`0`) and count (`data.Length`)